berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -157,6 +157,7 @@ class MapMarkerDetailView: UIView {
         verticalStack.addArrangedSubview(nameLabel)
         if (notesLabel.text?.count ?? 0) > 0 {
             verticalStack.addArrangedSubview(notesLabel)
+            nameLabel.rightAnchor.constraint(equalTo: closeButton.rightAnchor, constant: -10).isActive = true
```

Comment:
I think you should add this line to the `init` function above, where the other UI elements are laid out. Also, you probably want to constrain `nameLabel.rightAnchor` to `closeButton.leftAnchor` (instead of `rightAnchor`). As is, it still overlaps a bit:

<img width="834" alt="Screen Shot 2020-09-26 at 5 11 49 PM" src="https://user-images.githubusercontent.com/41145903/94352649-92fbec80-001c-11eb-8eac-eaa3792862fc.png">